.github/workflows/update.yaml

```diff
@@ -0,0 +1,40 @@
+name: update-files
+
+on:
+  workflow_call:
+    secrets:
+      AUTOMATION_TOKEN:
+        required: true
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: local checkout
+        uses: actions/checkout@v3
+      - name: checkout modules-template
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.AUTOMATION_TOKEN }}
+          repository: 'boldlink/test-repository2'
+          ref: 'feature/test-branch'
+          path: .tmp
+      - name: copy files
+        run: |
+          cp  -R ${GITHUB_WORKSPACE}/.tmp/.github/ ./
```

Comment:
I understand you are using this command to copy every file we want to update, how do we make sure it is the first workflow to run?

The command should be easier to read, can you make it a multiline one?

Reply:
This workflows does not run together with other workflows..It is set to run on a schedule.

Reply:
I understand now, what is the frequency?

Reply:
How do you auto-merge?